rasa/shared/nlu/training_data/util.py

```diff
@@ -50,6 +52,15 @@ def check_duplicate_synonym(
 
 
 def get_file_format(resource_name: Text) -> Text:
+    """
+    Get the file format of a resource. It supports both a folder and
+    a file, and tries to guess the format as follows:
```

Comment:
please use the google docstring format which also describe parameters / return values